lib/client.js

```diff
@@ -454,6 +454,20 @@ class Freshdesk {
 		makeRequest('GET', this._auth, `${this.baseUrl}/api/v2/companies/${id}`, null, null, cb)
 	}
 
+		/**
+	 * Search Companies.
+	 *
+	 * @see {@link https://developers.freshdesk.com/api/#search_companies}
+	 *
+	 * @category Companies
+	 *
+	 * @param  {String}  id  Company name
+	 * @param  {Freshdesk.requestCallback<Freshdesk.CompanyViewData>}  cb
+	 * Callback function
+	 */
+	searchCompany(name, cb) {
+		makeRequest("GET", this._auth, `${this.baseUrl}/api/v2/companies/autocomplete?name=${name}`, null, null, cb)
```

Comment:
you can pass the query string to `makeRequest`, for example: https://github.com/arjunkomath/node-freshdesk-api/blob/1af451bb1fb7b06df79a68b1bec3d1cdee437327/lib/client.js#L107

Reply:
Yup I thought about that, wasn't sure what method should be used since there was clearly quite a few different people working on this. I'll swap it out.